<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hex Coverage Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="currentSlide">
         <property name="title">
          <string>Slide Selector</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QComboBox" name="slideSelector"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="castersBox">
         <property name="title">
          <string>Caster Names</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="casterName2">
            <property name="text">
             <string>Caster One</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="casterName1">
            <property name="text">
             <string>Caster Two</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="castersShort">
            <property name="text">
             <string>One &amp; Two</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="twitterBox">
         <property name="title">
          <string>Twitter Handles</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="twitterHandles">
            <property name="text">
             <string>@CastOne &amp; @CastTwo</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gamesBox">
         <property name="title">
          <string>Game Wins</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="gameWinsTop">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="gameWinsBottom">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="roundBox">
         <property name="title">
          <string>Round Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="roundInformation">
            <property name="text">
             <string>Round X of Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playerRecord1">
            <property name="text">
             <string>A-B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playerRecord2">
            <property name="text">
             <string>C-D</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="eventBox">
         <property name="title">
          <string>Event Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="hashTag">
            <property name="text">
             <string>#HexEvent</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonBox">
      <property name="title">
       <string>Scene Control</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="startRound">
         <property name="text">
          <string>Start Round</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showBooth">
         <property name="text">
          <string>Show Booth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showSlide">
         <property name="text">
          <string>Show Current Slide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showGameScreen">
         <property name="text">
          <string>Show Game Screen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="endRound">
         <property name="text">
          <string>End Round</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
